config.json

```diff
@@ -1564,6 +1564,16 @@
       "topics": [
         "arrays"
       ]
-    }
+    },
+    {
+      "slug": "knapsack",
+      "uuid": "b0301d0b-d97a-4043-bd82-ba1edf8c1b16",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "algorithms"
+      ]
+    }    
```

Comment:
I had to pull down your branch and run `configlet fmt .` myself to figure this out: Travis is failing because there is trailing whitespace at Line 1577.

Reply:
Thanks for that, it passed Travis.